FinniversKit/Sources/SwiftUI Components/Button/SwiftUIIconButton.swift

```diff
@@ -16,7 +16,8 @@ public struct SwiftUIIconButton: View {
 
     public var body: some View {
         Image.init(isToggled ? style.iconToggled : style.icon)
-            .accessibility(addTraits: isToggled ? [.isSelected] : [])
+            .accessibilityRemoveTraits(.isImage)
+            .accessibility(addTraits: isToggled ? [.isButton, .isSelected] : [.isButton])
```

Comment:
This method is deprecated. We should use [`accessibilityAddTraits(_:)`](https://developer.apple.com/documentation/swiftui/view/accessibilityaddtraits(_:)) instead